### ISSUE TITLE
len(original_list) 50 len(detected_list) 1  test and detect length desn't match

### ISSUE THREAD

**Issue by zhangwang0537** [lyl8213/Plate_Recognition-LPRnet#16]

**Comment by immarioo** [comment]
em, I get the same problem.. do you know what is the problem?

**Comment by bluesy7585** [comment]
I have the same problem. using provided pretrained model
the inference outputs 'decoded' is a sparse tensor of dense_shape [50, 0] and values in 'log_prob' are all zero.

train the model from scratch using images in './train', it doesn't have the problem.
'decoded' is a sparse tensor of dense_shape [50, 7]

**Comment by immarioo** [comment]
yes, I trained the model from scratch and using this model to test, it's ok, maybe the model given by the author is broken.  thanks.
I trained the train images(~750 images) for 300 epochs and its test accuracy is about 50%, I am trying to do dataaug and train again.
